packages/core/contracts/test/WormholeMock.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.2;
+
+import '../governance/wormhole/IWormholeReceiver.sol';
+import '../governance/wormhole/IWormholeRelayer.sol';
+import 'hardhat/console.sol';
```

Comment:
Remove console since it's used for debugging purpose.

Reply:
removed